backend/test/mocks/crunchbase/domain-airthium.com.json

```diff
@@ -0,0 +1 @@
+{"metadata":{"version":31,"www_path_prefix":"https://www.crunchbase.com","api_path_prefix":"https://api.crunchbase.com","image_path_prefix":"http://public.crunchbase.com/t_api_images/"},"data":{"paging":{"total_items":1,"number_of_pages":1,"current_page":1,"sort_order":"custom","items_per_page":100,"next_page_url":null,"prev_page_url":null,"key_set_url":null,"collection_url":"https://api.crunchbase.com/v3.1/organizations","updated_since":null},"items":[{"type":"OrganizationSummary","uuid":"f21dbee02fb520b65b91da08a3abeebc","properties":{"permalink":"airthium","api_path":"organizations/airthium","web_path":"organization/airthium","api_url":"https://api.crunchbase.com/v3.1/organizations/airthium","name":"Airthium","stock_exchange":null,"stock_symbol":null,"primary_role":"company","short_description":"Airthium makes low cost, zero emission, 30kW+ Thermodynamic energy storage systems, as well as high efficiency diesel generators","profile_image_url":"http://public.crunchbase.com/t_api_images/v1503423190/hbk5jk03rluo7u9mwpig.png","domain":"airthium.com/","homepage_url":"http://www.airthium.com/","facebook_url":null,"twitter_url":"https://twitter.com/airthium","linkedin_url":null,"city_name":"Palaiseau","region_name":"Ile-de-France","country_code":"France","created_at":1503423197,"updated_at":1539755818}}]}}
```

Comment:
maybe we should display the json formatted, i can be useful for reference to better understand the code?

Reply:
Good call